### PR TITLE
Topo Naming Accumulated Fix

### PR DESCRIPTION
**Pull request by realthunder**
Forum thread https://forum.freecadweb.org/viewtopic.php?f=44&t=71687&sid=c92e0d2bbd2d0fadc0b6bb42a7128889&start=10#p626675

### CODE REVIEW

File: src/Mod/Part/App/FeatureExtrusion.cpp [hunk 1]
```diff
@@ -72,8 +72,11 @@ Extrusion::Extrusion()
     ADD_PROPERTY_TYPE(Symmetric, (false), "Extrude", App::Prop_None, "If true, extrusion is done in both directions to a total of LengthFwd. LengthRev is ignored.");
     ADD_PROPERTY_TYPE(TaperAngle, (0.0), "Extrude", App::Prop_None, "Sets the angle of slope (draft) to apply to the sides. The angle is for outward taper; negative value yields inward tapering.");
     ADD_PROPERTY_TYPE(TaperAngleRev, (0.0), "Extrude", App::Prop_None, "Taper angle of reverse part of extrusion.");
-    ADD_PROPERTY_TYPE(InnerTaperAngle,(0.0), "Extrude", App::Prop_None, "Taper angle of inner holes.");
-    ADD_PROPERTY_TYPE(InnerTaperAngleRev,(0.0), "Extrude", App::Prop_None, "Taper angle of the reverse part for inner holes.");
+    ADD_PROPERTY_TYPE(TaperInnerAngle,(0.0), "Extrude", App::Prop_None, "Taper angle of inner holes.");
+    ADD_PROPERTY_TYPE(TaperInnerAngleRev,(0.0), "Extrude", App::Prop_None, "Taper angle of the reverse part for inner holes.");
+    ADD_PROPERTY_TYPE(AutoTaperInnerAngle,(true), "Extrude", App::Prop_None,
```
Comment (donovaly) [inline src/Mod/Part/App/FeatureExtrusion.cpp:77]: This is a new option and should therefore not be part of the toponaming branch.

Please only commit what is absolutely necessary for toponaming. All further features are of course very welcome, but please sort them out to each a separate PR for master.

**Review by donovaly — CHANGES_REQUESTED** [review]

**Comment by berndhahnebach** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7514">pipeline status</a> for feature branch [PR_7514](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7514). Pipeline [649698444](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/649698444) was triggered at [32c75d1](https://github.com/FreeCAD/FreeCAD/pull/7514/commits/32c75d1ddb218a2d4f8c091819019e7a472c7a7a). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by berndhahnebach** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7514">pipeline status</a> for feature branch [PR_7514](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7514). Pipeline [649710732](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/649710732) was triggered at [22c0f02](https://github.com/FreeCAD/FreeCAD/pull/7514/commits/22c0f0253b2b2d5ce08b046f1822db7e898d847b). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by donovaly** [comment]
Many thanks!

I would merge, but get this error:
```
Check failure on line 70 in src/Gui/Application.cpp
GitHub Actions / Checks

'GuiInitScript.h' file not found [clang-diagnostic-error]

```
see https://github.com/FreeCAD/FreeCAD/pull/7514/commits/22c0f025

Can you please have a look?

**Comment by realthunder** [comment]
We can ignore this error, because the file is generated during the build [here](https://github.com/FreeCAD/FreeCAD/blob/cef5e6f4abe33f446354c01187bafecf71922ba0/src/Gui/CMakeLists.txt#L237).

**Comment by berndhahnebach** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7514">pipeline status</a> for feature branch [PR_7514](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7514). Pipeline [650059861](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/650059861) was triggered at [d2bef89](https://github.com/FreeCAD/FreeCAD/pull/7514/commits/d2bef8900b2719e191c6dcb8d87a55b29ad27a39). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by donovaly** [comment]
many thanks! I merged this now.

Before announcing, can you please review the toponaming branch if it contains some new features that are not necessary for toponaming? Just to be sure we have the most lean code change.

**Pull request merged**